util/raclgen.py

```diff
@@ -55,6 +55,7 @@ def main():
 
     template = """
 <%doc>
+  Note: The paramter names must be generated identically in toplevel.sv.tpl
```

Comment:
Mis-spelling of "parameter". (While you're fixing it, can you change the commit title too?)